### PR TITLE
Make clown clumsy.

### LINKED ISSUE #1407: Clowns must be clumsy
Guns should explode in their hands.

### PR DESCRIPTION
**Pull request by F77F**
Fixes #1407

### CODE REVIEW

**Comment by AJCM-git** [comment]
At last an useful PR

**Comment by F77F** [comment]
> At last an useful PR

yes please upvote emoji for awareness this is very important

File: Content.Server/GameObjects/Components/Weapon/Ranged/Barrels/ServerRangedBarrelComponent.cs [hunk 1]
```diff
@@ -206,6 +209,29 @@ private void Fire(IEntity shooter, GridCoordinates target)
...
+            if(shooter.HasComponent<ClumsyComponent>())
+            {
+
+                soundSystem.PlayAtCoords("/Audio/Items/bikehorn.ogg",
+                Owner.Transform.GridPosition, AudioParams.Default, 5);
+
+                soundSystem.PlayAtCoords("/Audio/Weapons/Guns/Gunshots/bang.ogg",
+                Owner.Transform.GridPosition, AudioParams.Default, 5);
+
+                var health = shooter.GetComponent<DamageableComponent>();
```
Comment (PJB3005) [inline Content.Server/GameObjects/Components/Weapon/Ranged/Barrels/ServerRangedBarrelComponent.cs:222]: Maybe put this behind a `TryGetComponent`?

File: Content.Server/GameTicking/GameTicker.cs [hunk 2]
```diff
@@ -774,6 +774,10 @@ private async void SpawnPlayer(IPlayerSession session, string jobId = null, bool
             AddManifestEntry(character.Name, jobId);
             AddSpawnedPosition(jobId);
             EquipIdCard(mob, character.Name, jobPrototype);
+            if(jobPrototype.ID == "Clown")
```
Comment (PJB3005) [inline Content.Server/GameTicking/GameTicker.cs:777]: Yeah this isn't gonna fly.

Best solution here is to allow custom C# logic to be specified from the job prototype and then doing this there.

Comment (F77F) [reply Content.Server/GameTicking/GameTicker.cs:777]: ah, nvm then.

File: Content.Server/GameObjects/Components/Weapon/Ranged/Barrels/ServerRangedBarrelComponent.cs (hunk 1 above)
Comment (F77F) [reply Content.Server/GameObjects/Components/Weapon/Ranged/Barrels/ServerRangedBarrelComponent.cs:222]: i did that at first actually.

**Comment by SweptWasTaken** [comment]
Rip

**Pull request closed without merging**